client/src/pages/Therapist/CreateProgram/AddDescription.js

```diff
@@ -13,28 +13,42 @@ import {
 import validate from '../../../validation/schemas/programme';
 
 const { Row, Col } = Grid;
-const { Textarea } = Inputs;
+const { Textarea, BasicInput } = Inputs;
+const initialStates = {
+  description: '',
+  title: '',
+};
+const AddDescription = ({
+  navFunctions,
+  setParentDescription,
+
+  clientId,
+}) => {
+  const [desc, setDesc] = useState(initialStates);
 
-const AddDescription = ({ navFunctions, setParentDescription, clientId }) => {
-  const [description, setDescription] = useState(null);
+  // const [title, setTitle] = useState('');
   const [submitAttempt, setSubmitAttempt] = useState(false);
   const [descriptionError, setDescriptionError] = useState(null);
-
+  const [titleError, setTitleError] = useState(null);
```

Comment:
insted of making errors vars for each field, add a single errors objects thats will make the code more cleaner and less amount of
```suggestion
  const [errors, serErros] = useState({});
```